README.md

```diff
@@ -1,53 +1,129 @@
+    2. [Doker run](#tc13)
+        1. [Enviroment informations](#tc131)
+        2. [Using docker cli](#tc132)
+        3. [Using docker compose](#tc133)
+    3. [Local run](#tc12)
+2. [Debug and development](#tc2)
+    1. [Database schemas](#tc21)
 
-Clone this repo with checkout to branch `master-2.0`. Run the `DiscordBot` project. :bangbang: Before running remember to add to the main direcotry file `token.txt` with discord bot token.
+## Running bot <a id="tc1"></a>
 
-## Running in docker
+This part is about how to run bot depending on enviroment used.
 
-### Using docker compose
+### Imporatnt informations <a id="tc11"></a>
 
-Container can be run using provided docker compose file provided example provided in `docker-compose-template.yaml`, or from the example below.
+Before trying to run you need to aquire token from [Discord](https://discord.com/developers/applications).
+
+Project supports use of [`Seq`](https://datalust.co/seq) server which can be run locally or remotly for storing logs.
+
+### Docker run <a id="tc13"></a>
+
+Here steps for use of deployable version of DiscordBot. Default Docker Hub image could be found [here](https://hub.docker.com/r/seker212/discord_bot)
+
+#### Enviroment informations <a id="tc131"></a>
+
+Running in docker requires setting important evnviromental variables as well as some volumes for files that should be accessible after shutdown of container.
+
+Variables marked with `Seq server` are referenced to different image for running Seq toghether with DiscordBot
+
+Enviroment variables:
+
+| Name |Value|Required | Purpose  |
+|----------|:-------------:|:----:|----|
+| TOKEN |  string (long chain) | X |  Discord bot token can't be run without it. |
+| SEQ_URL |  string (url) |  | Seq application url, local or remote |
+| ACCEPT_EULA |  Y or N (single char) |  | `Seq server` container acceptance of eula if run with DiscordBot together|
+
+Volumes:
+
+| Name |      Container Path      | Required | Purpose |
+|----------|:-------------:|:----:|-----|
+| Audio |  `/app/audio` | X | All sounds files are stored here |
+| Data |  `/app/data` | | Database file is stored here with all servers configurations |
+| Seq/logs |  `/data` | | `Seq server` container data folder for logs storage if run with DiscordBot together|
+
+#### Using docker cli  <a id="tc132"></a>
+
+Pulling and running container could be done using command below, simply place the TOKEN and name, and set the volume path in PWD.
+
+```bash
+docker run --rm -d -e "TOKEN=<discord app client secret here>" --name <name> -v <PWD>/audio:/app/audio seker212/discord_bot:2.0
+```
+
+#### Using docker compose <a id="tc133"></a>
+
+Use provided [`docker-compose-template.yaml`](docker-compose-template.yaml) and rename it to `docker-compose.yaml` or based on example below and enviroment variables create your own `docker-compose.yaml` file. Replace required values and run by using `docker compose up -d` command for Docker v2 or `docker-compose up -d` for v1.
```

Comment:
"[...] or based on example below and enviroment variables create [...]" - I don't like the construction of this sentence. Maybe "[...] or create [...] based on example below."?